### ISSUE TITLE
Many missing dependencies when building 

### ISSUE THREAD

**Issue by WaterSibilantFalling** [ungoogled-software/ungoogled-chromium-debian#274]
Note: this is an unsupported Debian version, testing, rather than stable (Bullseye) or sid. 
So you might want to auto-close this ticket

```
uname -a
Linux  5.15.0-2-amd64 #1 SMP Debian 5.15.5-1 (2021-11-26) x86_64 GNU/Linux
Debian Bookworm (testing)
```

A large number of dependencies are missing. 

```
sudo mk-build-deps -i debian/control
dpkg-buildpackage: info: source package ungoogled-chromium-build-deps
dpkg-buildpackage: info: source version 95.0.4638.69-1
dpkg-buildpackage: info: source distribution unstable
dpkg-buildpackage: info: source changed by Equivs Dummy Package Generator <root@Isis>
dpkg-buildpackage: info: host architecture amd64

      <<<  snip >>>


dpkg-buildpackage: info: binary-only upload (no source included)

The package has been created.
Attention, the package has been created in the current directory,
not in ".." as indicated by the message above!
Selecting previously unselected package ungoogled-chromium-build-deps.
(Reading database ... 378097 files and directories currently installed.)
Preparing to unpack ungoogled-chromium-build-deps_95.0.4638.69-1_all.deb ...
Unpacking ungoogled-chromium-build-deps (95.0.4638.69-1) ...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Correcting dependencies...Starting pkgProblemResolver with broken count: 1
Starting 2 pkgProblemResolver with broken count: 1
Investigating (0) ungoogled-chromium-build-deps:amd64 < 95.0.4638.69-1 @iU K Nb Ib >
Broken ungoogled-chromium-build-deps:amd64 Depends on lld-11:amd64 < none @un H >
  Considering lld-11:amd64 1 as a solution to ungoogled-chromium-build-deps:amd64 -2            <------------
  Removing ungoogled-chromium-build-deps:amd64 rather than change lld-11:amd64
Done
 Done
Starting pkgProblemResolver with broken count: 0
Starting 2 pkgProblemResolver with broken count: 0
Done
The following packages will be REMOVED:
  ungoogled-chromium-build-deps
0 upgraded, 0 newly installed, 1 to remove and 1 not upgraded.
1 not fully installed or removed.
After this operation, 10.2 kB disk space will be freed.
Do you want to continue? [Y/n] n
Abort.
(Reading database ... 378101 files and directories currently installed.)
Removing ungoogled-chromium-build-deps (95.0.4638.69-1) ...
mk-build-deps: Unable to install all build-dep packages


```

so I killed the process as soon as the `ungoogled-chromium-build-deps.dep`  was built, and then did
   `dpkg-i ungoogled-chromium-build-deps.dep `
to get a list of the missing dependencies, viz

```

 sudo dpkg -i *.deb
(Reading database ... 378997 files and directories currently installed.)
Preparing to unpack ungoogled-chromium-build-deps_95.0.4638.69-1_all.deb ...
Unpacking ungoogled-chromium-build-deps (95.0.4638.69-1) over (95.0.4638.69-1) ...
dpkg: dependency problems prevent configuration of ungoogled-chromium-build-deps:
 ungoogled-chromium-build-deps depends on dh-exec; however:
  Package dh-exec is not installed.
 ungoogled-chromium-build-deps depends on python-is-python3; however:
  Package python-is-python3 is not installed.
 ungoogled-chromium-build-deps depends on ninja-build; however:
  Package ninja-build is not installed.
 ungoogled-chromium-build-deps depends on gperf; however:
  Package gperf is not installed.
 ungoogled-chromium-build-deps depends on libnss3-dev; however:
  Package libnss3-dev is not installed.
 ungoogled-chromium-build-deps depends on libkrb5-dev; however:
  Package libkrb5-dev is not installed.
 ungoogled-chromium-build-deps depends on libgbm-dev; however:
  Package libgbm-dev is not installed.
 ungoogled-chromium-build-deps depends on libva-dev; however:
  Package libva-dev is not installed.
 ungoogled-chromium-build-deps depends on libpulse-dev; however:
  Package libpulse-dev is not installed.
 ungoogled-chromium-build-deps depends on libpci-dev; however:
  Package libpci-dev is not installed.
 ungoogled-chromium-build-deps depends on libopus-dev; however:
  Package libopus-dev is not installed.
 ungoogled-chromium-build-deps depends on libwebp-dev; however:
  Package libwebp-dev is not installed.
 ungoogled-chromium-build-deps depends on libflac-dev; however:
  Package libflac-dev is not installed.
 ungoogled-chromium-build-deps depends on libre2-dev; however:
  Package libre2-dev is not installed.
 ungoogled-chromium-build-deps depends on libsnappy-dev; however:
  Package libsnappy-dev is not installed.
 ungoogled-chromium-build-deps depends on liblcms2-dev; however:
  Package liblcms2-dev is not installed.
 ungoogled-chromium-build-deps depends on libopenjp2-7-dev; however:
  Package libopenjp2-7-dev is not installed.
 ungoogled-chromium-build-deps depends on libjsoncpp-dev; however:
  Package libjsoncpp-dev is not installed.

dpkg: error processing package ungoogled-chromium-build-deps (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 ungoogled-chromium-build-deps

```

and then installed them 

`sudo aptitude install python-is-python3 ninja-build gperf libnss3-dev libkrb5-dev libgbm-dev libva-dev  libpulse-dev libpci-dev libopus-dev libwebp-dev  libflac-dev  libre2-dev  libsnappy-dev liblcms2-dev libopenjp2-7-dev libjsoncpp-dev lld-dev clang-11  `

and then it worked

```

The package has been created.
Attention, the package has been created in the current directory,
not in ".." as indicated by the message above!
(Reading database ... 379813 files and directories currently installed.)
Preparing to unpack ungoogled-chromium-build-deps_95.0.4638.69-1_all.deb ...
Unpacking ungoogled-chromium-build-deps (95.0.4638.69-1) over (95.0.4638.69-1) ...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Starting pkgProblemResolver with broken count: 0
Starting 2 pkgProblemResolver with broken count: 0
Done
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up ungoogled-chromium-build-deps (95.0.4638.69-1) ...
======  How can you help?  (doc: https://wiki.debian.org/how-can-i-help ) ======

-----  Show old opportunities as well as new ones: how-can-i-help --old  -----


```

**Comment by ghost** [comment]
New release in progress, this is no longer relevant.
